provers/sp1/guest/Cargo.toml

```diff
@@ -37,18 +37,29 @@ path = "src/benchmark/bn254_mul.rs"
 [dependencies]
 raiko-lib = { path = "../../../lib", features = ["std", "sp1"] }
 sp1-zkvm ={ git = "https://github.com/succinctlabs/sp1.git", branch = "main" }
-revm-precompile = { git = "https://github.com/taikoxyz/revm.git", branch = "v35_taiko_v2", default-features = false, features = [
+revm-precompile = { git = "https://github.com/taikoxyz/revm.git", branch = "v35-taiko", default-features = false, features = [
     "taiko",
     "std",
     "c-kzg",
 ] }
-revm-primitives = { git = "https://github.com/taikoxyz/revm.git", branch = "v35_taiko_v2", default-features = false }
+revm-primitives = { git = "https://github.com/taikoxyz/revm.git", branch = "v35-taiko", default-features = false }
 sha2-v0-10-8 = { git = "https://github.com/sp1-patches/RustCrypto-hashes", package = "sha2", branch = "v0.10.8" }
 sp1-precompiles = { git = "https://github.com/succinctlabs/sp1.git", branch = "main"}
 harness-core = { path = "../../../harness/core" }
 harness = { path = "../../../harness/macro", features = ["sp1"]}
 substrate-bn = "0.6.0"
-
+alloy-consensus = { git = "https://github.com/ceciliaz030/alloy", branch = "175_4e22b9e-cycle", optional = true, features = [
+    "serde",
+] }
 [patch.crates-io]
 # Cant's use ZkOp for keccak since it's not precompile
+ecdsa = { git = "https://github.com/taikoxyz/signatures.git", branch = "0.16.9", package = "ecdsa", default-features = false, features = ["der"] }
 tiny-keccak = { git = "https://github.com/sp1-patches/tiny-keccak", branch = "patch-v2.0.2" }
+secp256k1 = { git = "https://github.com/CeciliaZ030/rust-secp256k1", branch = "sp1-patch", features = ["recovery", "sp1"]}
```

Comment:
should we fork this repo into the taiko repo as well?

Reply:
u used alloy from ur repo as well 😅 doesn't hurt

Reply:
Which was a questionable decision :)

Similar to the problems we had with revm, we just need to have some PR system going on to make sure that updates on those dependencies are also reviewed and can be merged in when necessary, forking it as part of the taiko repo's has some benefits for that.

Same thing for the c-kzg/alloy/... forks we currently have. Ideally I think they should all be taiko forks.